code/modules/surgery/organs/tongue.dm

```diff
@@ -69,10 +69,18 @@
 /obj/item/organ/tongue/lizard/handle_speech(datum/source, list/speech_args)
 	var/static/regex/lizard_hiss = new("s+", "g")
 	var/static/regex/lizard_hiSS = new("S+", "g")
+	var/static/regex/lizard_kss = new(@"(\w)x", "g")
+	var/static/regex/lizard_kSS = new(@"(\w)X", "g")
+	var/static/regex/lizard_ecks = new(@"\bx([\-|r|R]|\b)", "g")
+	var/static/regex/lizard_eckS = new(@"\bX([\-|r|R]|\b)", "g")
```

Comment:
`|` is not needed within regex character sets, this only applies capture groups, instead it's just capturing `|` characters twice. You also do not need to escape `-`. `[-rR]` should work fine.

Reply:
i believe the original individual who coded it said they felt neurotic about the letters not capturing capitalized cases correctly. I will test the function without them

Reply:
![image](https://user-images.githubusercontent.com/62388554/214698698-2ba851e3-97dc-4d6f-8a9f-bdf591dd4fc7.png)
addressed